package.json

```diff
@@ -43,19 +43,19 @@
         "docs": "typedoc"
     },
     "devDependencies": {
-        "@biomejs/biome": "1.7.1",
-        "@commitlint/cli": "19.3.0",
-        "@commitlint/config-conventional": "19.2.2",
-        "@vitest/coverage-v8": "1.5.1",
-        "husky": "9.0.11",
-        "lint-staged": "15.2.2",
+        "@biomejs/biome": "1.8.3",
```

Comment:
:question: Is doing this in the same commit as the `feat` going to affect anything?